python/ray/rllib/__init__.py

```diff
@@ -27,6 +35,24 @@ def _register_all():
 _register_all()
 
 __all__ = [
-    "PolicyGraph", "TFPolicyGraph", "CommonPolicyEvaluator", "SampleBatch",
-    "AsyncVectorEnv", "MultiAgentEnv", "VectorEnv", "ServingEnv",
+    # rllib.env
+    "AsyncVectorEnv", "MultiAgentEnv", "ServingEnv", "VectorEnv", "EnvContext",
+
+    # rllib.evaluation
+    "EvaluatorInterface", "PolicyEvaluator", "PolicyGraph", "TFPolicyGraph",
+    "TorchPolicyGraph", "SampleBatch", "MultiAgentBatch", "SampleBatchBuilder",
+    "MultiAgentSampleBatchBuilder", "SyncSampler", "AsyncSampler",
+    "compute_advantages", "collect_metrics",
+
+    # rllib.models
+    "ActionDistribution", "ActionDistribution", "Categorical",
+    "DiagGaussian", "Deterministic", "ModelCatalog", "Model",
+    "Preprocessor", "FullyConnectedNetwork", "LSTM",
+
+    # rllib.optimizers
+    "PolicyOptimizer", "AsyncSamplesOptimizer", "AsyncGradientsOptimizer",
+    "SyncSamplesOptimizer", "SyncReplayOptimizer", "LocalMultiGPUOptimizer",
+
+    # rllib.utils
+    "Filter", "FilterManager", "PolicyClient", "PolicyServer"
```

Comment:
Hmm this is a lot more than seems necessary, why not stick to the current set of imports and add just a couple more if needed?